### PR TITLE
Heading restructure

### PR DESCRIPTION
**Pull request by MHaton**
As discussed with @pomegranited , have had a first pass at amending headings within the templates to allow for the heading structure change from Eucalyptus. For the most part it's just bumping them down a level, with one being bumped down from 2 to 4 (as unless it's not the element I'm thinking of, its heading level was too high to begin with).

Note that there's references to other H2+H3's etc. contained within things like the python tests that I've not touched, so presumably those will fail and also need updating, I just didn't want to mess with anything I don't understand. I've also not amended the apros theme CSS. Basically, half the task still to do.

### CODE REVIEW

**Comment by pomegranited** [comment]
@MHaton If you could go ahead and update the apros.css and the python tests, that would be great.

I'll run the tests and make sure they pass, and we'll test this in the Apros environment too.

**Comment by MHaton** [comment]
@pomegranited Updated what I think is the last few places h2 & old h3's appear. Needs testing, my workbench has stopped working.

**Comment by pomegranited** [comment]
Thank you for this, @MHaton !  I've asked to get this into next week's sprint, but it might end up being the week after that.

**Comment by bradenmacdonald** [comment]
@MHaton With the ongoing release of Ficus we're pretty busy this week so we've scheduled a review of this PR for the week of March 6. Sorry for the delay.

File: problem_builder/tests/integration/test_titles.py [hunk 1]
```diff
@@ -145,9 +145,9 @@ def test_all_the_things(self):
...
                     pb_element = self.go_to_view()
                     if expected_title:
-                        h3 = pb_element.find_element_by_css_selector('h3')
+                        h4 = pb_element.find_element_by_css_selector('h4')
                         self.assertEqual(h3.text, expected_title)
                     else:
-                        # No <h3> element should be present:
-                        all_h3s = pb_element.find_elements_by_css_selector('h3')
-                        self.assertEqual(len(all_h3s), 0)
+                        # No <h4> element should be present:
```
Comment (pomegranited) [inline problem_builder/tests/integration/test_titles.py]: 2 lines above needs to be updated too:

```
self.assertEqual(h4.text, expected_title)
```

Comment (MHaton) [inline problem_builder/tests/integration/test_titles.py]: Sure thing, will do this on Monday :)

**Comment by MHaton** [comment]
Version number updated (now conflicting), test sorted

Take it away @pomegranited  :)

**Comment by pomegranited** [comment]
Tagged as v2.6.8.

**Pull request merged**